test/test_composite.py

```diff
@@ -10,45 +10,302 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Class to test composite experiments."""
+"""Test suite for composite experiments.
 
-from test.fake_backend import FakeBackend
-from test.fake_experiment import FakeExperiment
+This test assumes some simple but enough general virtual experiment.
```

Comment:
In the future we will have additional tests for composite experiments, which will do other things (and run experiments different from the ones described here). Therefore this documentation's location is not in the file's description.

Reply:
Make sense. Please find the update in https://github.com/Qiskit/qiskit-experiments/pull/380/commits/b7b112809ab50ea9f82f19be39cf983239af88ae

I keep the module docstring as general as possible while specific details are moved to the test class docstring. Note that the purpose of module docstring is to tell developers not to import actual experimental classes for the composite experiment test, because it makes debugging quite complicated and also makes the test unstable.